src/Requests/BatchResponseContent.php

```diff
@@ -0,0 +1,142 @@
+<?php
+/**
+ * Copyright (c) Microsoft Corporation.  All Rights Reserved.
+ * Licensed under the MIT License.  See License in the project root
+ * for license information.
+ */
+
+
+namespace Microsoft\Graph\Core\Requests;
+
+use Microsoft\Kiota\Abstractions\Serialization\Parsable;
+use Microsoft\Kiota\Abstractions\Serialization\ParseNode;
+use Microsoft\Kiota\Abstractions\Serialization\SerializationWriter;
+
+/**
+ * Class BatchResponseContent
+ *
+ * @package Microsoft\Graph\Core
+ * @copyright 2022 Microsoft Corporation
+ * @license https://opensource.org/licenses/MIT MIT License
+ * @link https://developer.microsoft.com/graph
+ */
+class BatchResponseContent implements Parsable
```

Comment:
Perhaps we can support the methods `getResponseById(string $responseId)` whuch is standard across all the other languages https://learn.microsoft.com/en-us/graph/sdks/batch-requests?tabs=csharp